BeardedSpice/MediaStrategies/iHeartRadioStrategy.m

```diff
@@ -0,0 +1,82 @@
+//
+//  iHeartRadioStrategy.m
+//  BeardedSpice
+//
+//  Created by Coder-256 on 2/7/16.
+//  Copyright © 2016 BeardedSpice. All rights reserved.
+//
+
+#import "iHeartRadioStrategy.h"
+
+@implementation iHeartRadioStrategy
+
+- (id)init {
+    self = [super init];
+    
+    if (self) {
+        predicate = [NSPredicate predicateWithFormat:@"SELF LIKE[c] '*iheart.com*'"];
+    }
+    
+    return self;
+    
+}
+-(BOOL) accepts:(TabAdapter *)tab
+{
+    return [predicate evaluateWithObject:[tab URL]];
+}
+
+-(BOOL)isPlaying:(TabAdapter *)tab
+{
+    NSNumber *value = [tab executeJavascript:@"(document.querySelectorAll('[aria-label=\"Stop\"]').length > 0)"];
+    return [value boolValue];
+}
+
+-(NSString *) toggle
+{
+    return @"(function(){if (document.querySelectorAll('[aria-label=\"Stop\"]').length > 0) {document.querySelector('[aria-label=\"Stop\"]').click();} else {var plays = document.querySelectorAll('[aria-label=\"Play Station\"]'); plays[plays.length-1].click();}})()";
+}
+
+
+-(NSString *) pause
+{
+    return @"(function(){document.querySelector('[aria-label=\"Stop\"]').click();})()";
+}
+
+- (NSString *)next {
```

Comment:
As mentioned in df1fd25, this changes stations if skip is unavailable (as is the case with live radio).